packages/react-app/theme.js

```diff
@@ -1,10 +1,14 @@
 import { extendTheme } from "@chakra-ui/react";
 
-const config = {
-  initialColorMode: "light",
-  useSystemColorMode: false,
-};
-
-const theme = extendTheme({ config });
```

Comment:
inlined the `config` in `extendTheme` itself, lol I almost wasted an hour figuring why fonts were not working, I was adding `fonts` key  to `config` thinking that it was used directly passed  in `extendTheme` but lol after debugging noticed config is separate key and `fonts` is separate key to `extendTheme` obj 

but please fee free to revert it !